src/utils/CMakeLists.txt

```
@@ -30,13 +30,42 @@ if(Git_FOUND)
     OUTPUT_VARIABLE OPENDIGITIZER_GIT_DESCRIBE
     OUTPUT_STRIP_TRAILING_WHITESPACE)
 endif()
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/version/version.cpp.in ${CMAKE_BINARY_DIR}/generated_version/version.cpp
-               @ONLY)
+
 if(Git_FOUND)
-  set_property(
-    DIRECTORY
-    APPEND
-    PROPERTY CMAKE_CONFIGURE_DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/../../.git/HEAD")
+  # cmake-format: off
+  set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS
+          "${CMAKE_CURRENT_SOURCE_DIR}/../../.git/HEAD"
+          "${CMAKE_CURRENT_SOURCE_DIR}/../../.git/index"
+          "${CMAKE_CURRENT_SOURCE_DIR}/../../.git/refs/heads/master"
```

I haven't dug into why we depend on exactly these files, but we don't have a master branch
```suggestion
          "${CMAKE_CURRENT_SOURCE_DIR}/../../.git/refs/heads/main"
```